src/frontend/Sider/Behandling/Inngangsvilkår/Stønadsperioder/Stønadsperioder.tsx

```diff
@@ -144,7 +147,11 @@ const Stønadsperioder: React.FC = () => {
                                             formState.errors.stønadsperioder &&
                                             formState.errors.stønadsperioder[indeks]
                                         }
-                                        erLeservisning={!behandlingErRedigerbar || !redigerer}
+                                        erLeservisning={
+                                            !behandlingErRedigerbar ||
+                                            !redigerer ||
```

Comment:
Vet den ikke er lagt til nå men er det egentlig vits å ha `!redigerer` her? 
Er egentlig nok for å vite om det er leservisning å sjekke `steg !== inngangsvilkår` (på en penere måte)

Reply:
Ja, det er mulig

Reply:
Jo, den må fortsatt være med. I hvert fall så lenge vi har knappen for "Rediger stønadsperioder".

Reply:
ja sorry tenkte ikke på at det var stønadsperiodene